config/base/crds/kafka.banzaicloud.io_kafkaclusters.yaml

```diff
@@ -3916,13 +3942,41 @@ spec:
                     description: ExternalListenerConfig defines the external listener
                       config for Kafka
                     properties:
+                      accessMethod:
+                        description: accessMethod defines the method which the external
+                          listener is exposed through. Two types are supported LoadBalancer
+                          and NodePort. The recommended and default is the LoadBalancer.
+                          NodePort should be used in Kubernetes environments with
+                          no support for provisioning Load Balancers.
+                        enum:
+                        - LoadBalancer
+                        - NodePort
```

Comment:
would it make sense to also add `None` as an option here? to disable external access altogether?

Reply:
Removing all external listeners would disable external access altogether. kafka-operator should remove all K8s that correspond to an external listener when that is removed from the CR.